src/libsyntax/attr.rs

```diff
@@ -453,26 +573,38 @@ pub fn requests_inline(attrs: &[Attribute]) -> bool {
 
 /// Tests if a cfg-pattern matches the cfg set
 pub fn cfg_matches(cfgs: &[P<MetaItem>], cfg: &ast::MetaItem,
-                   sess: &ParseSess, features: Option<&Features>)
+                   sess: &ParseSess, fs: Option<&Features>)
```

Comment:
I'm pretty sure this doesn't adhere to what `rustfmt` would do, anyway. It should be:

```rust
pub fn cfg_matches(cfgs: &[P<MetaItem>],
                   cfg: &ast::MetaItem,
                   sess: &ParseSess,
                   features: Option<&Features>)
                   -> bool {
```